frame/clearing-house/src/tests/add_margin.rs

```diff
@@ -47,14 +50,24 @@ fn deposit_supported_collateral_succeeds() {
 			run_to_block(1);
 			let account = ALICE;
 			let asset = USDC;
-			let amount: Balance = 1_000_u32.into();
+			let amount: Balance = 1_000;
 
-			let before = AccountsMargin::<Runtime>::get(&account).unwrap_or_default();
+			let before = (
+				AccountsMargin::<Runtime>::get(&account).unwrap_or_default(),
+				<AssetsPallet as Inspect<AccountId>>::balance(USDC, &ALICE),
```

Comment:
maybe just change 'ALICE' to 'account'

Reply:
same with line 63

Reply:
I think I'll introduce `BOB` pretty soon when I start testing the flow of funding payments, so perhaps it's good to maintain a distinctive name